src/components/FileSelector/styled/index.js

```diff
@@ -186,6 +213,17 @@ export const StyledButtonIcon = styled(ButtonIcon)`
     svg {
         pointer-events: none;
     }
+
+    ${props =>
+        props.variantValue === 'multiline' &&
+        `
+        position: absolute;
+        top: -14px;
```

Comment:
why we need to pass `-14px`?
It is a bad practice since it makes the code more complex and difficult for developer's understanding

what's happen with the button if on the container styles example we add ` height: 500,`
`const containerStyles = {
    height: 500,
};`

@TahimiLeonBravo please, review it

Reply:
ok, never try here again negative values

Reply:
we can't use this `top: -14px` here because when we change the height of the container we get it 
<img width="508" alt="Screen Shot 2020-05-21 at 8 19 52 PM" src="https://user-images.githubusercontent.com/32083556/82621362-89f79c00-9ba0-11ea-90e3-cd5c00ef78d6.png">
@blakmetall  we need find a better solution here, we should follow always the best-practices and be consitent